plato/draw/vispy/__init__.py

```diff
@@ -18,6 +18,7 @@
 
 from .Scene import Scene
 
+from .Box import Box
```

Comment:
Please keep the primitives in alphabetical order, so this goes after `Arrows2D`.

Reply:
Ordered.